h2o-r/h2o-package/R/automl.R

```diff
@@ -183,6 +192,17 @@ h2o.automl <- function(x, y, training_frame,
   }
   build_control$nfolds <- nfolds
   
+  # Update build_control with balance_classes & related args
+  if (balance_classes == TRUE) {
+    build_control$balance_classes <- balance_classes
+  }
+  if (!is.null(class_sampling_factors)) {
+    build_control$class_sampling_factors <- class_sampling_factors
+  }
+  if (max_after_balance_size != 5) {
```

Comment:
Does max_after_balance_size get filled in to be 5.0 if a user doesn't change the parameter?

Reply:
No, if the user does not change it from 5.0, we don't send it.  That's what we've been doing with the other args, so it's consistent.

Reply:
So it essentially has no effect. Got it.